pingen_env/models/res_company.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api,fields,models
+from odoo.addons.server_environment import serv_config
+
+class ResCompany(models.Model):
+    _inherit = ["res.company"]
+
+    pingen_clientid = fields.Char(compute='_compute_pingen_env',
+                            required=False,
+                            readonly=True)
+    pingen_client_secretid = fields.Char(compute='_compute_pingen_env',
+                               required=False,
+                               readonly=True)
+    pingen_organization = fields.Char(compute='_compute_pingen_env',
+                            required=False,
+                            readonly=True)
+    pingen_staging = fields.Boolean(compute='_compute_pingen_env',
+                            required=False,
+                            readonly=True)
+    pingen_webhook_secret = fields.Char(compute='_compute_pingen_env',
+                                       required=False,
+                                       readonly=True)
+
+    @api.depends()
+    def _compute_pingen_env(self):
+        global_section_name = "pingen"
+        for company in self:
+            # default vals
+            config_vals = {}
+            if serv_config.has_section(global_section_name):
+                config_vals.update((serv_config.items(global_section_name)))
+            custom_section_name = global_section_name + "." + company.name
+            if serv_config.has_section(custom_section_name):
+                config_vals.update(serv_config.items(custom_section_name))
```

Comment:
If you put a `my_attribute = False` here it could be the `False` will be interpreted as a trueish string, and will set the `pingen_staging` field to True. Maybe use `str2bool` from `odoo.tools` to handle this for this particular field.